tests/api/conftest.py

```
@@ -8,10 +8,87 @@
+        username: str = "ssouser",
+        auth_method: str = "sso",
+        must_change_password: bool = False,
+        is_active: bool = True,
+        link_sso_identity: bool = True,
+    ) -> User:
+        user = user_crud.create(
+            db_session,
+            obj_in=UserCreate(
+                username=username,
+                email=f"{username}@example.com",
+                password="localpassword123",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)